test/arrays/bradc/formalActualArrayArgMatch.chpl

```diff
@@ -0,0 +1,45 @@
+use BlockDist;
```

Comment:
I think you need a suppressif for this test for `--fast` 

See `test/arrays/bradc/slices/outOfBoundsSlice.suppressif` for an example of the text we use for other tests that are explicitly testing runtime checks